frontend/core-ui/src/modules/products/components/product-command-bar/ProductMerge.tsx

```
@@ -0,0 +1,432 @@
+  const { t } = useTranslation('product');
+  const [productA, productB] = products;
+
+  const initValues = useMemo(() => {
+    const values: Record<string, any> = {};
+    for (const field of MERGE_FIELDS) {
+      const a = field.getValue(productA);
+      const b = field.getValue(productB);
+      values[field.key] = hasValue(a) ? a : b;
+    }
+    return values;
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)